client/src/components/Search/Filters/FilterTypes/DateFilter/DateFilter.js

```diff
@@ -0,0 +1,172 @@
+        const {onRemove} = this.props
+        await this.handleTypeChange(null);
+        onRemove()
+    }
+    render() {
+        const { filterMode, dateFilterStart, dateFilterEnd, params } = this.state;
+        const decade = params.decade
+        const { decades } = this.props;
+        const minimumDate = new Date(1, 0);
+        minimumDate.setFullYear(1);
+        const currentDate = new Date();
+        const Slider = require('rc-slider');
+        const createSliderWithTooltip = Slider.createSliderWithTooltip;
+        const Range = createSliderWithTooltip(Slider.Range);
+        const sliderStyle = { width: 400, margin: 25 };
+        let dateFilter = (<div/>);
+        const marks = {
+            
+            1870: '1870\'s or Earlier',
+            1900: '1900\'s',
+            1930: '1930\'s',
+            1960: '1960\'s',
+            1990: '1990\'s',
+            2020: 'Present'
+        }
+        if (filterMode === 'range') {
+            dateFilter = (
+                <div className="d-flex align-items-center">
+                    <span className="mr-2">Start Date</span>
+                    <DatePicker
+                        selected={dateFilterStart}
+                        onChange={(date) => this.handleRangeChange('start', date)}
+                        minDate={minimumDate}
+                        maxDate={currentDate}
+                    />
+                    <span className="ml-3 mr-2">End Date</span>
+                    <DatePicker
+                        selected={dateFilterEnd}
+                        onChange={(date) => this.handleRangeChange('end', date)}
+                        minDate={minimumDate}
+                        maxDate={currentDate}
+                    />
+                </div>
+            );
+        } else if (filterMode === 'decade') {
+            dateFilter = (
+                <div className="d-flex align-items-center">
+                    <span className="mr-2">Monuments or memorials created in the</span>
+                    <Form.Control as="select" className="min-width-select" onChange={event => this.handleDateFilter('decade', event.target.value)} value={decade}>
+                        <option value="null">None</option>
+                        {decades.map(decade => (
+                            <option value={decade} key={decade}>{decade}s</option>
+                        ))}
+                    </Form.Control>
+                </div>
+            );
+        }else if (filterMode === 'slider') {
+            dateFilter = (
+                <div className="d-flex align-items-center">
+                    <span className="mr-2">Active in</span>
+                    <div style = {sliderStyle}>
+                        <Range allowCross={false} min={1870} max={2020} step={10} defaultValue={[1870, 1960]} marks = {marks} 
+                        handleStyle= {{borderColor: '#42b883', backgroundColor: '#42b883', borderRadius: '0%', width: '6px', height: '18px'}}/>
```

Comment:
not sure the difference between style and handleStyle, but move this definition to a css file if possible